linux/cpu_hot_add_basic/generate_cpu_hot_add_list.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 VMware, Inc.
```

Comment:
I think we can move this task file to common folder as it is been used by both Linux and Windows.

Reply:
These common tasks are test case specific, so I think we can keep it there, if there is better way for common used tasks in test cases for both Linux and Windows, we can change it then. Thanks.